webapp/src/monaco.tsx

```diff
@@ -839,6 +842,7 @@ export class Editor extends srceditor.Editor {
                     let currPos = monacoEditor.editor.getPosition();
                     let cursor = model.getOffsetAt(currPos)
                     let insertText = snippetPrefix ? `${snippetPrefix}.${snippet}` : snippet;
+                    insertText = addNamespace ? `${/([^\.]+)/.exec(element.namespace)[0]}.${insertText}` : insertText;
```

Comment:
Let's move this regex (and the one below) into a function. Also, the parentheses seem unnecessary since you are using the full match anyways